user_guide_src/source/installation/running.rst

```diff
@@ -426,6 +426,87 @@ Setting Environment
 
 See :ref:`Handling Multiple Environments <environment-nginx>`.
 
+
+.. _deployment-to-shared-hosting-services:
+
+*************************************
+Deployment to Shared Hosting Services
+*************************************
+
+.. important::
+    **index.php** is no longer in the root of the project! It has been moved inside
+    the **public** folder, for better security and separation of components.
+
+    This means that you should configure your web server to "point" to your project's
+    **public** folder, and not to the project root.
+
+Specifying the Document Root
+============================
+
+The best way is to set the document root to the **public** folder in the server
+configuration::
+
+    └── example.com/ (project folder)
+        └── public/  (document root)
+
+For example, if you use cPanel, see:
+
+- `How to change the document root of the primary domain <https://support.cpanel.net/hc/en-us/articles/360055211213-How-to-change-the-document-root-of-the-primary-domain>`_
```

Comment:
A shared hosting subscriber how can access To WHM as root (step 4)?
Assuming it has access to WHM. Where is the menu he explained(step 5)?

![Screenshot 2024-02-17 064944](https://github.com/codeigniter4/CodeIgniter4/assets/9530214/54ed0826-b158-4ffe-9f9a-1686b06f8df5)

Reply:
See https://docs.cpanel.net/knowledge-base/accounts/how-to-log-in-to-your-server-or-account/#how-to-access-whm

Reply:
How do I log in to my cPanel account as root? – cPanel
https://support.cpanel.net/hc/en-us/articles/360051754554-How-do-I-log-in-to-my-cPanel-account-as-root

Reply:
Perhaps we are not normally allowed to log in as root. If so, this method is not generally available. 
In that case, it should not be mentioned in the documentation.

Reply:
I had a call with the hosting company.
The possibility of logging for shared hosting users is provided only  through cPanel NOT WHM.

Regarding the change of Document Root, Answer was, considering that the host is shared, it is **NOT** possible to change the Document Root.

![Screenshot 2024-02-17 081936](https://github.com/codeigniter4/CodeIgniter4/assets/9530214/e17e9df4-a89d-4f1f-9673-b729c8aad489)

Reply:
@datamweb Thank you for investigation! I've removed the URLs for cPanel.